receiver/prometheusreceiver/metrics_receiver_helper_test.go

```diff
@@ -110,11 +112,11 @@ var (
 )
 
 type testData struct {
-	name           string
-	pages          []mockPrometheusResponse
-	attributes     pdata.AttributeMap
-	useOpenMetrics bool
-	validateFunc   func(t *testing.T, td *testData, result []*pdata.ResourceMetrics)
+	name             string
+	pages            []mockPrometheusResponse
+	attributes       pdata.AttributeMap
+	skipValidScrapes bool
```

Comment:
```suggestion
	validateScrapes bool
```

This should probably be phrased in the affirmative and better describe what it affects.  I initially thought that this would cause the tests to skip testing of valid scrapes and only test invalid scrapes, but instead it is controlling whether the function called `getValidScrapes()` is invoked.  Another way to phrase it might be `skipScrapeValidation`.